src/app/pages/website/info/information/information.component.html

```diff
@@ -1,16 +1,11 @@
 <littil-title>Informatie</littil-title>
 
 <littil-content-container>
-    <div class="grid grid-cols-12 gap-x-4 my-12">
-      <article class="col-span-12 md:col-span-8">
-        <p>Platform LITTIL is op dit moment op zoek naar
-          <a routerLink="/info/schools">scholen</a>,
-          <a routerLink="/info/it-specialists">IT specialisten</a> en
-          <a routerLink="/info/sponsors">sponsors</a>.
-          Mocht u interesse hebben dan vragen wij om één van deze pagina’s te bezoeken en contact met ons op te nemen.
-        </p>
-      </article>
-
-    </div>
+  <p>Platform LITTIL is op dit moment op zoek naar
+    <a routerLink="/info/schools">scholen</a>,
+    <a routerLink="/info/it-specialists">IT specialisten</a> en
+    <a routerLink="/info/sponsors">sponsors</a>.
+    Mocht u interesse hebben dan vragen wij om één van deze pagina’s te bezoeken en contact met ons op te nemen.
```

Comment:
Can you check the routerLinks. Don't seem to work.

Reply:
Fixed